CI is broken on main

This broke main
 
```
Error: /home/runner/work/opentelemetry-collector-contrib/opentelemetry-collector-contrib/processor/tailsamplingprocessor/metrics.go:93:16: undefined: obsreport
```

@jmsnll @jpkrohling would you be able to do a quick fix? Or should I revert the PR and we can address this more calmly?

_Originally posted by @mx-psi in https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/26724#issuecomment-1737519704_

--- a/processor/tailsamplingprocessor/metrics.go
+++ b/processor/tailsamplingprocessor/metrics.go
@@ -90,7 +90,7 @@ func samplingProcessorMetricViews(level configtelemetry.Level) []*view.View {
 	}
 
 	countGlobalTracesSampledView := &view.View{
-		Name:        obsreport.BuildProcessorCustomMetricName(metadata.Type, statCountGlobalTracesSampled.Name()),
+		Name:        processorhelper.BuildCustomMetricName(metadata.Type, statCountGlobalTracesSampled.Name()),
 		Measure:     statCountGlobalTracesSampled,
 		Description: statCountGlobalTracesSampled.Description(),
 		TagKeys:     []tag.Key{tagSampledKey},

